exporter/trace/cloudtrace.go

```diff
@@ -142,21 +138,13 @@ func WithTimeout(t time.Duration) func(o *options) {
 }
 
 // WithDisplayNameFormatter sets the way span's display names will be
-// generated from SpanSnapshot
+// generated from ReadOnlySpan
 func WithDisplayNameFormatter(f DisplayNameFormatter) func(o *options) {
 	return func(o *options) {
 		o.DisplayNameFormatter = f
 	}
 }
 
-// WithDefaultTraceAttributes sets the attributes that will be appended
```

Comment:
It should still be possible to keep this functionality by modifying the Google Cloud Trace proto at export time, even if it's no longer possible to modify the OTel `ReadOnlySpan`.

Reply:
Yes, I see. I will add this to the `ConvertSpan` function.

Reply:
I have change `protoFromReadOnlySpan` function signature to accept a list of default trace attributes. This values will be added to the attribute set within the function.
What do you think?

Reply:
This approach makes sense, thank you!